.gitignore

```diff
@@ -1,3 +1,6 @@
+#Adversarial Robustness Toolbox files
+datasets/
```

Comment:
What is the purpose of the directory `datasets`?

Reply:
a sorry, yep that was an omission. It was just temporary while I was working on my side. I've removed this change thanks for pointing it out